src/peft/tuners/fourierft/config.py

```diff
@@ -185,6 +185,15 @@ class FourierFTConfig(PeftConfig):
         },
     )
 
+    alpha: float = field(
```

Comment:
IIUC, `alpha` would take precedence over both `n_frequency` and `n_frequency_pattern`, right? This should be documented. Furthermore, it would make sense to check in the `__post_init__` if a user has set both `alpha` and `n_frequency`/`n_frequency_pattern` and raise an error in that case.

Let's then add a test for this error to `test_initialization.py`?